luxjs/grid/grid.js

```diff
@@ -96,8 +119,8 @@
                     if (!col.hasOwnProperty('filter'))
                         column.enableFiltering = false;
 
-                    if (column.field === 'id')
-                        column.cellTemplate = gridDefaults.wrapCell('<a ng-href="{{grid.appScope.objectUrl(COL_FIELD)}}">{{COL_FIELD}}</a>');
+                    if (column.field === repr_field)
+                        column.cellTemplate = gridDefaults.wrapCell('<a ng-href="{{grid.appScope.objectUrl(row.entity.id)}}">{{COL_FIELD}}</a>');
```

Comment:
the metadata object contains two entries, ``id`` and ``repr``.
Both can be any thing, usually
```json
{
   "id": "id",
   "repr": "name"
}
but it could well be that
```json
{
    "id": "username",
    "repr": "full_name"
}
```
One thing to bear in mind, the id is not necessarily the database id.